css/app.css

```diff
@@ -1,3 +1,913 @@
+/* reset&global */
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  font-family: "Montserrat", sans-serif;
+}
+
 body {
-  background-color: blue;
+  background-color: #f5f5f5;
+  max-width: 1920px;
+  margin: 0 auto;
+}
+
+ul {
+  list-style-type: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+section {
+  margin: 5.3125rem 0;
+}
+
+.flex {
+  display: flex;
+}
+
+.grid {
+  display: grid;
+}
+
+.align-center {
+  align-items: center;
+}
+
+.section-title {
+  margin-bottom: 1.25rem;
+}
+
+.section-title a {
+  color: #f42;
+}
+
+.footer-breakline {
+  display: none;
+}
+
+/* navbar-top */
+
+.nav-logo {
+  display: none;
+  justify-content: center;
+  background: #f42;
+  background: linear-gradient(to bottom right, #c09, #f42 85%);
+  padding: 0 1.25rem;
+}
+
+.nav-logo img {
+  height: 1rem;
+  width: 4.125rem;
+}
+
+.navbar-container {
+  flex-grow: 1;
+  height: 81px;
+}
+
+.nav-top {
+  height: 35%;
+  background-color: black;
+  justify-content: space-between;
+  line-height: 2rem;
+  padding: 0 0.9375rem;
+}
+
+.nav-top-left,
+.nav-top-right {
+  justify-content: space-around;
+}
+
+.nav-main-search i {
+  color: rgb(190, 190, 190);
+  padding: 0.875rem 1.25rem;
+  border-left: 1px solid rgba(190, 190, 190, 0.5);
+  transform: rotateY(180deg);
+}
+
+.latest-funding-top-right a {
+  padding: 0 0.625rem;
+  color: #f42;
+  color: inherit;
+}
+
+.latest-funding-footer a {
+  color: #555;
+}
+
+.main-article-social,
+.main-article-social a {
+  font-size: 0.75rem;
+  color: #fff9;
+}
+
+.main-article-subarticles a {
+  font-size: 0.8125rem;
+  color: rgb(99, 99, 99);
+}
+
+.latest-deals-header > a {
+  color: #f42;
+}
+
+.nav-top-left .active {
+  color: white;
+}
+
+/* navbar-main */
+.nav-main {
+  height: 65%;
+  justify-content: space-between;
+  padding-left: 0.9375rem;
+  background-color: #fff;
+}
+
+.nav-main-hidden {
+  height: 100%;
+  display: none;
+}
+
+.nav-main-left {
+  justify-content: space-around;
+}
+
+.nav-main-search-sandwitch {
+  display: none;
+}
```

Comment:
Same style definition of selector on line 50, group selectors with the same style definition, or create a new class.